### ISSUE TITLE
MS Windows failure (probably related to Graphics-Primitive-Driver-GD-0.01)

### ISSUE THREAD

**Issue by kmx** [gphat/chart-clicker#44]
Hi,

I have experienced the following failure on strawberry perl 5.22.0 (MS Windows)

```
"D:\strawberry32\perl\bin\perl.exe" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib\lib', 'blib\arch')" t/*.t t/series/*.t t/xt/*.t
t/add-data.t .................... ok
t/axis-datetime.t ............... ok
t/axis-division-rounded.t ....... ok
t/axis.t ........................ ok
t/basic.t ....................... ok
t/context.t ..................... ok
t/data-dataset.t ................ ok
t/data-marker.t ................. ok
t/data-range.t .................. ok
Usage: GD::Image::filledRectangle(image, x1, y1, x2, y2, color) at D:/strawberry32/perl/site/lib/Graphics/Primitive/Driver/GD.pm line 451.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 2.
t/datetime.t .................... 
Dubious, test returned 255 (wstat 65280, 0xff00)
All 2 subtests passed 
t/decoration-color_allocator.t .. ok
t/decoration-glass.t ............ ok
t/decoration-grid.t ............. ok
t/decoration-legend.t ........... ok
t/decoration-markeroverlay.t .... ok
t/decoration-plot.t ............. ok
t/decoration.t .................. ok
t/drawing-colorallocator.t ...... ok
Usage: GD::Image::filledRectangle(image, x1, y1, x2, y2, color) at D:/strawberry32/perl/site/lib/Graphics/Primitive/Driver/GD.pm line 451.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 2.
t/example.t ..................... 
Dubious, test returned 255 (wstat 65280, 0xff00)
All 2 subtests passed 
t/invisible.t ................... ok
t/release-pod-syntax.t .......... skipped: these tests are for release candidate testing
t/release-synopsis.t ............ skipped: these tests are for release candidate testing
t/renderer-area.t ............... ok
t/renderer-bar.t ................ ok
t/renderer-base.t ............... ok
t/renderer-bubble.t ............. ok
t/renderer-candlestick.t ........ ok
t/renderer-line.t ............... ok
t/renderer-pie.t ................ ok
t/renderer-point.t .............. ok
t/same-axes-ctx.t ............... ok
t/series/highlow.t .............. ok
t/series/series.t ............... ok
t/series/size.t ................. ok
t/simple-ctx.t .................. ok
t/xt/pod.t ...................... ok

Test Summary Report
-------------------
t/datetime.t                  (Wstat: 65280 Tests: 2 Failed: 0)
  Non-zero exit status: 255
  Parse errors: No plan found in TAP output
t/example.t                   (Wstat: 65280 Tests: 2 Failed: 0)
  Non-zero exit status: 255
  Parse errors: No plan found in TAP output
Files=36, Tests=338, 36 wallclock secs ( 0.19 usr +  0.05 sys =  0.23 CPU)
Result: FAIL
```

**Comment by dionys** [comment]
Fixed at gphat/graphics-primitive-driver-gd#1.

**Comment by gphat** [comment]
Released!
```
POSTing upload for Graphics-Primitive-Driver-GD-0.02.tar.gz
PAUSE add message sent ok [200]
```
